fastlane/Fastfile

```diff
@@ -180,7 +180,7 @@ platform :ios do
     )
   end
 
-  desc "Add a tag for the current build number and push to repo."
+  desc "Add a release tag for the latest beta and push to repo."
```

Comment:
FYI this is the same description as the next lane on line 192. Could we also add a "This is very similar to..." sentence to one of these to differentiate?

Reply:
Updated, thanks.